webpack-modules/README.md

```diff
@@ -0,0 +1,31 @@
+# WebExtension Webpack Example
+A minimal example of how to use npm modules from within a WebExtension.
+The example package used by this extension is `left-pad`, an essential package
+in almost any situation.
+
+## What it does
+This setup takes a background and a content script entrypoint and outputs them
```

Comment:
Left pad gets included once as background script and once as content script, thus two build targets. I did that to show, that you can use webpack both for content and background scripts.